src/Symfony/Component/Notifier/Bridge/Brevo/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+CHANGELOG
+=========
+
+6.4
+---
+
+* Added the bridge
```

Comment:
```suggestion
* Add the bridge
```

We can add an info about a replacement in the README; no need for it in the changeling file. From a code POV this is simply a new bridge